### DISCUSSION TITLE
Grafana Dashboard with job runs / status ?!

### DISCUSSION THREAD

**Discussion by Viral** [windmill-labs/windmill#4520] [Questions]
hello !! 

looking for path of least resistance to get ALL of the job status into a Grafana dashboard for our ops team ?! any suggestions ?

thank you !!

**Comment by alp82** [comment]
I'm very interested in this too.

**Comment by Viral** [comment]
we went with querying the API - https://app.windmill.dev/openapi.html#tag/job/GET/w/{workspace}/jobs/list
